src/main/resources/site/parts/entryLinks/entryLinks.xml

```diff
@@ -0,0 +1,23 @@
+<part>
+    <display-name>Innganger med ikoner</display-name>
+    <description></description>
+    <form>
+        <item-set name="entryLinks">
+            <occurrences minimum="0" maximum="4" />
```

Comment:
Kanskje greit med en <label>Inngang</label> eller noe sånn at det blir litt mer oversiktlig i part config for brukeren :)